src/linear_algebra/alternating.lean

```diff
@@ -274,6 +288,31 @@ lemma map_congr_perm [fintype ι] (σ : equiv.perm ι) :
   g v = (equiv.perm.sign σ : ℤ) • g (v ∘ σ) :=
 by { rw [g.map_perm, smul_smul], simp }
 
+/-- If the arguments are linearly dependent then the result is `0`.
+
+TODO: Can the `division_ring` requirement be relaxed? -/
+lemma map_linear_dependent
```

Comment:
```suggestion
lemma map_not_linear_independent
```
I know this name looks weird... but I think the consistency is worth it in the long run.

Reply:
I was aiming for consistency with https://leanprover-community.github.io/mathlib_docs/linear_algebra/linear_independent.html#linear_dependent_iff